client/package.json

```diff
@@ -11,7 +11,9 @@
     "@testing-library/jest-dom": "^5.16.2",
     "@testing-library/react": "^12.1.4",
     "@testing-library/user-event": "^13.5.0",
+    "bootstrap": "^5.1.3",
     "react": "^17.0.2",
+    "react-bootstrap": "^2.3.1",
```

Comment:
Can you just remove this package. And I can merge

Reply:
Okay, I will do that now.